samples/c/common/opencv_c_wrapper/bmp_reader.c

```diff
@@ -96,7 +96,7 @@ int readBmpImage(const char* fileName, BitMap* image) {
     int i;
     int image_height = image->height;
     for (i = 0; i < image_height; i++) {
-        unsigned int storeAt = image->infoHeader.height < 0 ? i : (unsigned int)image_height - 1 - i;
+        unsigned int storeAt = image->infoHeader.height < 0 ? (unsigned int)i : (unsigned int)image_height - 1 - i;
```

Comment:
I suppose all arguments here are `int`, should we use this type instead of casting to `unsigned` one?